public/components/visualization/text2vega.ts

```diff
@@ -108,7 +108,8 @@ export class Text2Vega {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const escapeField = (json: any, field: string) => {
       if (json[field]) {
-        if (typeof json[field] === 'string') {
+        // Only escape field which name is 'field'
+        if (typeof json[field] === 'string' && field === 'field') {
```

Comment:
can we have a unit test to cover this?

Reply:
Yes, unit tests for t2viz will come in a separate PR